lib/Serialization/Serialization.cpp

```diff
@@ -3409,6 +3410,69 @@ void Serializer::writeDecl(const Decl *D) {
     break;
   }
 
+  // TODO: Revisit serialization.
+  // Create `CallLayout`? Not all fields from `FuncLayout` are necessary:
+  // `call` declarations cannot be static.
+  case DeclKind::Call: {
+    auto fn = cast<CallDecl>(D);
+    verifyAttrSerializable(fn);
+
+    auto contextID = addDeclContextRef(fn->getDeclContext());
+
+    unsigned abbrCode = DeclTypeAbbrCodes[FuncLayout::Code];
```

Comment:
TODO: Create a `CallLayout` that is more specialized than `FuncLayout` (like `AccessorLayout`), dropping some fields like static spelling.

Reply:
Deferred for now. `CallDecl` has only two less fields than `FuncDecl` (static location and static spelling.)